config/routes.rb

```diff
@@ -62,15 +62,18 @@
   resources :partnering_organizations do
     member do
       post :generate_new_password_email
+      patch 'approve'
+      get 'subscriptions'
```

Comment:
This one isn't necessary anymore because we got rid of that whole `id` thing up there^